### ISSUE TITLE
Desktop - Error trying to notarize DMG

### ISSUE THREAD

**Issue by ScottPierce** [JetBrains/compose-jb#959]
I'm getting the following errors when trying to notarize a DMG with Apple. Seems like maybe the packaging process is not signing some files that should be signed:

```
{
severity: "error",
code: null,
path: "My_App-1.0.0.dmg/My App.app/Contents/app/sqlite-jdbc-3.34.1-371b212b7663f42133e45d49e82ad30.jar/org/sqlite/native/Mac/x86_64/libsqlitejdbc.jnilib",
message: "The binary is not signed.",
docUrl: null,
architecture: "x86_64"
},
{
severity: "error",
code: null,
path: "My_App-1.0.0.dmg/My App.app/Contents/app/sqlite-jdbc-3.34.1-371b212b7663f42133e45d49e82ad30.jar/org/sqlite/native/Mac/x86_64/libsqlitejdbc.jnilib",
message: "The signature does not include a secure timestamp.",
docUrl: null,
architecture: "x86_64"
}
```

**Comment by AlexeyTsvetkov** [comment]
Should be resolved by #966

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6344">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
